IPython/sphinxext/ipython_directive.py

```diff
@@ -138,7 +138,13 @@
 
 # Third-party
 from docutils.parsers.rst import directives
-from sphinx.util.compat import Directive
+import sphinx
+# sphinx.version_info available since 1.2, and we require 1.3
+# sphinx.util.compat deprecated at 1.6 and removed at 1.7
+if sphinx.version_info < (1, 5, 7):
+    from sphinx.util.compat import Directive
+else:
+    from docutils.parsers.rst import Directive
```

Comment:
In fact at Sphinx 1.3, the sphinx.util.compat module simply contained:

    from docutils.parsers.rst import Directive  # noqa

and there was no monkey-patched done. Thus I feel doing this directly from IPython is safe, and arguably testing for Sphinx version is superfluous.